File: test/fixtures/output/java/asynchttp/compression.java
AsyncHttpClient client = new DefaultAsyncHttpClient();
client.prepare("GET", "http://mockbin.com/har")
  .setHeader("accept-encoding", "deflate, gzip, br")
  .execute()
  .toCompletableFuture()
  .thenAccept(System.out::println)
  .join();

client.close();

File: test/fixtures/output/php/http2/compression.php
<?php

$client = new http\Client;
$request = new http\Client\Request;

$request->setRequestUrl('http://mockbin.com/har');
$request->setRequestMethod('GET');
$request->setHeaders([
  'accept-encoding' => 'deflate, gzip, br'
]);

$client->enqueue($request)->send();
$response = $client->getResponse();

echo $response->getBody();